pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -2601,6 +2601,11 @@ in
     checkInputs = [
       self.fzf-lua
     ];
+
+    nvimSkipModules = lib.optionals stdenv.hostPlatform.isDarwin [
+      # FIXME: Fzf-lua address in use error we disabled in `luaPackages` check is also here
```

Comment:
Is it me or this comment does not make sense?

Reply:
Agree, but in nixpkgs, the issues tend to be forgotten more than comments. You can at least accidentally stumble on comments, but there is a huge amount of issues created every day

Reply:
That's why I recommend to reference the issue in the comment.
Issues often have a lot more detail (and discussion) than comments, so if you have a fixme/todo comment that references an issue, you get the best of both worlds